scripts/local_testnet/setup_time.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+source ./vars.env
+
+genesis_file=${@:$OPTIND+0:1}
+
+# Update future hardforks time in the EL genesis file based on the CL genesis time
+GENESIS_TIME=$(lcli pretty-ssz --spec $SPEC_PRESET --testnet-dir $TESTNET_DIR BeaconState $TESTNET_DIR/genesis.ssz | jq | grep -Po 'genesis_time": "\K.*\d')
+echo $GENESIS_TIME
+CAPELLA_TIME=$((GENESIS_TIME + (CAPELLA_FORK_EPOCH * 32 * SECONDS_PER_SLOT)))
```

Comment:
I've just noticed that this won't work for `minimal` preset (`SLOTS_PER_EPOCH: 8`) and `gnosis` preset (`SLOTS_PER_EPOCH: 16`). I'm not sure if there's a nice way of getting the preset values (it's available via [beacon API](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Config/getSpec) but requires beacon node running), perhaps we could just do a simple match the  `$SPEC_PRESET`?

```bash
get_spec_preset_value() {
  case "$SPEC_PRESET" in
    mainnet)   echo 32 ;;
    minimal)   echo 8  ;;
    gnosis)    echo 16 ;;
    *)         echo "Unsupported preset: $SPEC_PRESET" >&2; exit 1 ;;
  esac
}
```

Reply:
Great catch, thank you so much!
Fix in [f629452](https://github.com/sigp/lighthouse/pull/4749/commits/f62945227a447bb3969f3323dcd88f179f624b56)